src/Compilers/Core/VBCSCompiler/ServerDispatcher.cs

```diff
@@ -169,10 +168,6 @@ public void ListenAndDispatchConnections(string pipeName, TimeSpan? keepAlive, b
             Task<NamedPipeServerStream> listenTask = null;
             CancellationTokenSource listenCancellationTokenSource = null;
 
-            // If we aren't being asked to watch analyzer files then simple create a Task which never 
-            // completes.  This is the behavior of AnalyzerWatcher when files don't change on disk.
-            Task analyzerTask = watchAnalyzerFiles ? CompilerServerFileWatcher.CreateWatchFilesTask() : new TaskCompletionSource<bool>().Task;
```

Comment:
Question: I really never understood well enough what the analyzer file watcher was doing, but I am assuming the new behavior doesn't require having one anymore? I am asking this purely for my understanding.